google-cloud-storage/src/main/java/com/google/cloud/storage/transfermanager/ParallelUploadConfig.java

```diff
@@ -124,61 +125,82 @@ private static List<BlobWriteOption> applySkipIfExists(
     return writeOptsPerRequest;
   }
 
+  /**
+   * Builds an instance of ParallelUploadConfig.
+   *
+   * @see ParallelUploadConfig
```

Comment:
will this link to parallel upload config in javadoc?

Reply:
yep, it will render as `See Also: ParallelUploadConfig` and the ParallelUploadConfig bit will link to those javadocs. Similar to using link just not inline.